Sources/BraintreeCard/BTCard.swift

```diff
@@ -124,22 +158,10 @@ import Foundation
 
     private func buildCardDictionary(isGraphQL: Bool) -> [String: Any] {
         var cardDictionary: [String: Any] = [:]
-
-        if let number {
-            cardDictionary["number"] = number
-        }
-
-        if let expirationMonth {
-            cardDictionary[isGraphQL ? "expirationMonth" : "expiration_month"] = expirationMonth
-        }
-
-        if let expirationYear {
-            cardDictionary[isGraphQL ? "expirationYear" : "expiration_year"] = expirationYear
-        }
-
-        if let cvv {
-            cardDictionary["cvv"] = cvv
-        }
+        cardDictionary["number"] = number
+        cardDictionary[isGraphQL ? "expirationMonth" : "expiration_month"] = expirationMonth
+        cardDictionary[isGraphQL ? "expirationYear" : "expiration_year"] = expirationYear
```

Comment:
Are these fine with the CVV only init to pass in an empty string or should we do an `!number.isEmpty` check on these for example?

Reply:
We can add an empty validation, updated [6ae1adb](https://github.com/braintree/braintree_ios/pull/1443/commits/6ae1adb29b4ddc40f868683cfa1870aadc797b77)